boa/explorer.py

```diff
@@ -24,13 +24,14 @@ def _fetch_etherscan(
     if api_key is not None:
         params["apikey"] = api_key
 
-    for _ in range(num_retries):
+    for i in range(num_retries):
         res = SESSION.get(uri, params=params)
         res.raise_for_status()
         data = res.json()
         if not (data.get("status") == "0" and "rate limit" in data.get("result", "")):
             break
-        sleep(backoff_ms / 1000)
+        backoff_factor = 1.1**i  # 1.1**10 ~= 2.59
```

Comment:
better move the magic number to a constant or keyword arg

Reply:
hmm yea, i am planning to encapsulate the etherscan settings in a data structure, we can extract it into a kwarg then